invenio_users_resources/services/generators.py

```diff
@@ -13,9 +13,12 @@
 from functools import reduce
 from itertools import chain
 
+from flask_principal import RoleNeed
 from invenio_records_permissions.generators import Generator, UserNeed
 from invenio_search.engine import dsl
 
+from invenio_users_resources.permissions import moderation_action
```

Comment:
I would call it ``user_management_action`` so the it covers everything related to user management.